src/Features/LanguageServer/Protocol/Handler/CodeActions/CodeActionHelpers.cs

```diff
@@ -221,7 +217,7 @@ private static CodeAction GetNestedActionsFromActionSet(IUnifiedSuggestedAction
                 codeAction.Title, nestedActions.ToImmutable(), codeAction.IsInlinable, codeAction.Priority);
         }
 
-        private static async Task<ImmutableArray<UnifiedSuggestedActionSet>?> GetActionSetsAsync(
+        private static async Task<ImmutableArray<UnifiedSuggestedActionSet>> GetActionSetsAsync(
```

Comment:
📝 `AsyncTaskMethodBuilder<TResult>` knows how to cache `default(ImmutableArray<UnifiedSuggestedActionSet>?)`, but can't cache `default(ImmutableArray<UnifiedSuggestedActionSet>)`.

Reply:
i see.  i'm ok with that.  needing to to cache thsi value seems marginal.